.github/workflows/k8s-chart-upgrade-matrix.yaml

```
@@ -13,7 +13,7 @@ on:
         description: Rancher upgrade version
         required: true
         type: string
-        default: latest/devel/2.10
+        default: head/2.10
```

```suggestion
        default: head/2.12
```


rancher_version; default: latest/2.11.3